README.md

```diff
@@ -86,6 +78,12 @@ Itineraries
 
 Your key will be rate-limited to ~5 queries per minute.
 
+## Running the project
+
+The project is configured to run in docker using docker-compose to connect the client and server containers.
```

Comment:
It might be nice to include a `getting started` link for Docker - some candidates may have never really used it before ;)

Reply:
Good point. Done!